src/components/CardList.jsx

```diff
@@ -5,7 +5,7 @@ import CardItem from './CardItem';
 
 function CardList({ items }) {
   return (
-    <SimpleGrid minChildWidth="320px" spacing="5" alignContent="center">
+    <SimpleGrid minChildWidth="380px" spacing="5" alignContent="center">
```

Comment:
380px is too wide for smartphones, 320px is barely narrow enough to avoid side-scrolling. A better solution might be cropping titles in order to avoid having irregular cards

Reply:
Great! Now I make changes... what do you mean cropping titles? Less width?

Reply:
[Here](https://chakra-ui.com/docs/typography/text) in the Text docs, there is a isTruncated prop that should keep titles only 1 line long. This might do the trick